src/polar/PolarGrid.tsx

```diff
@@ -144,20 +135,41 @@ const ConcentricPath: React.FC<Props> = props => {
   );
 };
 
-export const PolarGrid = (props: Props) => {
-  const { outerRadius } = props;
-
+export const PolarGrid = ({
+  cx = 0,
+  cy = 0,
+  innerRadius = 0,
+  outerRadius = 0,
+  gridType = 'polygon',
+  radialLines = true,
+  ...props
+}: Props) => {
   if (outerRadius <= 0) {
     return null;
   }
 
   return (
     <g className="recharts-polar-grid">
-      <PolarAngles {...props} />
-      <ConcentricPath {...props} />
+      <PolarAngles
+        cx={cx}
+        cy={cy}
+        innerRadius={innerRadius}
+        outerRadius={outerRadius}
+        gridType={gridType}
+        radialLines={radialLines}
+        {...props}
+      />
+      <ConcentricPath
+        cx={cx}
+        cy={cy}
+        innerRadius={innerRadius}
+        outerRadius={outerRadius}
+        gridType={gridType}
+        radialLines={radialLines}
+        {...props}
```

Comment:
NIT: Since the de-structured values are props also, what do you think about renaming this to `otherProps`

Reply:
Yeah otherProps or restProps works - I always forget that spreading the rest doesn't contain the defaulted params. I can change on my next PR but will merge this for now